modules/mono/csharp_script.h

```diff
@@ -111,7 +111,7 @@ class CSharpScript : public Script {
 
 	void _clear();
 
-	bool _update_signals();
+	void load_script_signals(GDMonoClass *scrp_class);
```

Comment:
What does `scrp` mean? Script? Read-only parameters should start with `p_`, and I think `p_class` is good enough. If it needs to be more explicit, it would have to be `p_script_class`, not a mangled `scrp`.